### PR TITLE
feat: add headers for react-admin

### PR DESCRIPTION
**Pull request by markusahlstrand**
Added headers for range the way that react-admin wants it. It's not really correct as the returned range doesn't show the correct count.

### CODE REVIEW

**Comment by danielgent** [comment]
Dash is https://github.com/sesamyab/dash right? :smile: 

Quite a short name. How about `auth2-dashboard`?  `auth-dashboard`? (if we have admin dashboard)

These initial names burn us for a while afterwards *but* your plan might be to migrate the functionality into something else ASAP

**Review by danielgent — APPROVED** [review]
I can only see formatting changes :joy: 

I've spent 5 minutes or more scrolling up and down.

File: src/constants.ts [hunk 1]
```diff
@@ -9,8 +9,10 @@ export const headers = {
   accessControlAllowOrigin: "Access-Control-Allow-Origin",
   accessControlAllowMethod: "Access-Control-Allow-Methods",
   accessControlAllowCredentials: "Access-Control-Allow-Credentials",
+  accessControlExposeHeaders: "Access-Control-Expose-Headers",
   cacheControl: "cache-control",
   contentType: "content-type",
+  contentRange: "content-range",
```
Comment (danielgent) [inline src/constants.ts:15]: Cool cool ok. so this is for `auth0/nextjs` to talk to `auth2`. Do we need something similar to get the admin dashboard to work with the token service?

**Comment by markusahlstrand** [comment]
So, react-admin is a new thing I tried out.. Deployed here: https://auth-admin.vercel.sesamy.dev/. The project website is here: https://marmelab.com/react-admin

The have a default dataAdapter with pretty interesting defaults.. Think it makes sense to make it compatible.

**Comment by danielgent** [comment]
> So, react-admin is a new thing I tried out.. Deployed here: https://auth-admin.vercel.sesamy.dev/. The project website is here: https://marmelab.com/react-admin
> 
> The have a default dataAdapter with pretty interesting defaults.. Think it makes sense to make it compatible.

Cool cool this makes sense now. It uses `@auth0/auth0-spa-js` and a few other of its own libraries :+1:

**Pull request merged**